WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -256,8 +256,6 @@ public void onRefreshStarted() {
                             mSwipeToRefreshHelper.setRefreshing(false);
                             return;
                         }
-                        mDispatcher.dispatch(SiteActionBuilder.newRemoveWpcomAndJetpackSitesAction());
```

Comment:
Just curious, would that be a Jetpack site? I wonder how can a self-hosted site can be remotely removed while we still are on the SitePicker. Thanks!